extensions/blocks/eventbrite/edit.js

```diff
@@ -0,0 +1,379 @@
+						'jetpack'
+					) }
+				>
+					<div className="block-editor-block-styles">
+						{ embedTypes.map( this.renderEmbedTypeItem.bind( this ) ) }
+					</div>
+				</PanelBody>
+			</InspectorControls>
+		);
+	}
+
+	// Render embed types selection with previews, similar to block styles.
+	// https://github.com/WordPress/gutenberg/blob/wp/5.3/packages/block-editor/src/components/block-styles/index.js#L100
+	renderEmbedTypeItem( { alt, label, image, isActive, value } ) {
+		return (
+			<div
+				key={ value }
+				className={ classnames( 'block-editor-block-styles__item', {
+					'is-active': isActive,
+				} ) }
+				onClick={ () => this.setEmbedType( value ) }
+				onKeyDown={ event => {
+					if ( ENTER === event.keyCode || SPACE === event.keyCode ) {
+						event.preventDefault();
+						() => this.setEmbedType( value );
+					}
+				} }
+				role="button"
+				tabIndex="0"
+				aria-label={ label }
+			>
+				<div className="block-editor-block-styles__item-preview">
+					<div className="block-editor-block-preview__container">
+						<img src={ image } alt={ alt } />
+					</div>
+				</div>
+				<div className="block-editor-block-styles__item-label">{ label }</div>
+			</div>
+		);
+	}
+
+	renderBlockControls() {
+		return (
+			<BlockControls>
+				<Toolbar>
+					<IconButton
+						className="components-toolbar__control"
+						label={ __( 'Edit URL', 'jetpack' ) }
+						icon="edit"
+						onClick={ () => this.setState( { editingUrl: true } ) }
+					/>
+				</Toolbar>
+			</BlockControls>
+		);
+	}
+
+	renderEditEmbed() {
+		const { className } = this.props;
+		const { editedUrl } = this.state;
+		return (
+			<div className={ className }>
+				<Placeholder
+					label={ __( 'Eventbrite Checkout', 'jetpack' ) }
+					instructions={ __(
+						'Paste a link to an Eventbrite event to embed ticket checkout.',
+						'jetpack'
+					) }
+					icon={ <BlockIcon icon={ icon } showColors /> }
+				>
+					<form onSubmit={ this.setUrl }>
+						<input
+							type="url"
+							value={ editedUrl }
+							className="components-placeholder__input"
+							aria-label={ __( 'Eventbrite URL', 'jetpack' ) }
+							placeholder={ __( 'Enter an event URL to embed here…', 'jetpack' ) }
+							onChange={ event => this.setState( { editedUrl: event.target.value } ) }
+						/>
+						<Button isLarge type="submit">
+							{ _x( 'Embed', 'submit button label', 'jetpack' ) }
+						</Button>
+						{ this.cannotEmbed() && (
+							<p className="components-placeholder__error">
+								{ __( 'Sorry, this content could not be embedded.', 'jetpack' ) }
+								<br />
+								<Button isLarge onClick={ () => fallback( editedUrl, this.props.onReplace ) }>
+									{ _x( 'Convert to link', 'button label', 'jetpack' ) }
+								</Button>
+							</p>
+						) }
+					</form>
+
+					<div className="components-placeholder__learn-more">
+						<ExternalLink href="http://en.support.wordpress.com/wordpress-editor/blocks/eventbrite-block/">
```

Comment:
I assume this is already in our todo, but just in case, let's not forget to add some actual content to this support page.

Reply:
Yes, this is on my todo list--thanks for checking!